src/runtime_src/core/edge/sw_em/generic_pcie_hal2/halapi.cxx

```diff
@@ -1014,3 +1014,67 @@ xclLoadXclBinMeta(xclDeviceHandle handle, const xclBin *buffer)
   return 0;
```

Comment:
Copyright needs to be updated (line 3).  Being that we are part of AMD now, the following addition should be made.
`Copyright (C) Advanced Micro Devices, Inc.`